.gitignore

```diff
@@ -51,7 +51,7 @@ apache-rat-*.jar
 **.swp
 **/*ivy.xml
 .vagrant
-.example-image
+.toree-dev-image
```

Comment:
Will likely need to add `.binder-image` once the `touch` is added (per other comment).